tests/test_bukuDb.py

```diff
@@ -1684,6 +1684,12 @@ def test_load_firefox_database(bukuDb, firefox_db, add_pt):
     (True, ['+title', '-tags', 'description', 'index', 'uri'],
      ['javascript:void(1)', 'javascript:void(0)', 'http://slashdot.org',
       'http://example.com/', 'example.com/#', 'http://www.zażółćgęśląjaźń.pl/']),
+    (True, ['# tEst  ', '#invalid,tag', '-index'],
+     ['javascript:void(1)', 'javascript:void(0)', 'http://www.zażółćgęśląjaźń.pl/',
+      'http://slashdot.org', 'example.com/#', 'http://example.com/']),
+    (True, ['-#  teSt ', '-#invalid,tag', '-index'],
+     ['example.com/#', 'http://example.com/', 'javascript:void(1)',
+      'javascript:void(0)', 'http://www.zażółćgęśląjaźń.pl/', 'http://slashdot.org']),
```

Comment:
Testing collection sort with tags (tags with comma are invalid and therefore ignored)